docs/release/trg-1/trg-1-1.md

```diff
@@ -2,21 +2,40 @@
 title: TRG 1.01 - README.md
 ---
 
-| Status | Created      | Post-History |
-|--------|--------------|--------------|
-| Active | 07-Mar-2023  |              |
-| Draft  | 13-Sept-2022 |              |
+| Status | Created      | Post-History                             |
+|--------|--------------|------------------------------------------|
+| Active | 06-Dec-2023  | add detailed information for README file |
+| Active | 07-Mar-2023  |                                          |
+| Draft  | 13-Sept-2022 |                                          |
 
 ## Why
 
-A good `README.md` file is the starting point for everyone opening a repository. It should help them find all critical information in an easy way.
+A good `README.md` file is the starting point for everyone opening your repository. It should help them find all critical information in an easy way.
+A well-maintained README reflects positively on the overall professionalism of the project. It shows that the developers care about user experience and want to make it easy for others to understand and use their work.
```

Comment:
I would rephrase that a bit: A well-maintained README shows the high level of professionalism...

Reply:
[4090379](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/409037992f6ff3f7f93eb27e04e78edef5840b00)